File: app/src/dev/java/com/github/premnirmal/ticker/Analytics.java
package com.github.premnirmal.ticker;

import android.content.Context;

/**
 * Created by premnirmal on 4/15/15.
 */
public class Analytics {

    public static final String SCHEDULE_UPDATE_ACTION = "ScheduleUpdate";

    private static Analytics INSTANCE;

    private Analytics(Context context) {

    }

    static void init(Context context) {
        INSTANCE = new Analytics(context);
    }

    public static void trackUpdate(String action, String label) {

    }

    public static void trackWidgetUpdate(String action) {

    }

    public static void trackWidgetSizeUpdate(String value) {

    }

    public static void trackUI(String action, String label) {

    }

    public static void trackIntialSettings(String action, String label) {

    }

    public static void trackSettingsChange(String action, String label) {

    }

}

File: app/src/dev/kotlin/com/github/premnirmal/ticker/Analytics.kt
package com.github.premnirmal.ticker

import android.content.Context
import java.util.logging.Logger

/**
 * Created by premnirmal on 2/28/16.
 */
class Analytics private constructor(context: Context) {

  init {

  }

  companion object {

    @JvmField val SCHEDULE_UPDATE_ACTION = "ScheduleUpdate"

    private var INSTANCE: Analytics? = null

    internal fun init(context: Context) {
      INSTANCE = Analytics(context)
    }

    @JvmStatic fun trackUpdate(action: String, label: String) {

    }

    @JvmStatic fun trackWidgetUpdate(action: String) {

    }

    @JvmStatic fun trackWidgetSizeUpdate(value: String) {

    }

    @JvmStatic fun trackUI(action: String, label: String) {

    }

    @JvmStatic fun trackIntialSettings(action: String, label: String) {

    }

    @JvmStatic fun trackSettingsChange(action: String, label: String) {

    }

    @JvmStatic fun trackRateYes() {

    }

    @JvmStatic fun trackRateNo() {

    }
  }

}